test/functional/apps/dashboard/bwc_import.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import path from 'path';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const PageObjects = getPageObjects(['dashboard', 'header', 'settings', 'savedObjects', 'common']);
+  const dashboardExpect = getService('dashboardExpect');
+
+  describe('bwc import', function describeIndexTests() {
+    before(async function () {
+      await PageObjects.dashboard.initTests();
+      await PageObjects.settings.navigateTo();
+      await PageObjects.settings.clickKibanaSavedObjects();
+      await PageObjects.savedObjects.importFile(
+        path.join(__dirname, 'exports', 'dashboard_6_0_1.json')
```

Comment:
Is this a common location for an export?

Reply:
TBH, I am not sure what is the best practice. I followed another example I found where saved object import was used:
https://github.com/elastic/kibana/blob/master/test/functional/apps/management/exports/mgmt_import_objects.json